### PR TITLE
clarification on connected apps on business groups

### PR DESCRIPTION
**Pull request by lee3star**
clarification on connected apps on business groups

### CODE REVIEW

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 1]
```diff
@@ -47,3 +47,11 @@ Apps that remove scopes do not need to be whitelisted again.
 --
 Applications created by users in the same organization are marked as *internal apps* and do not need to be whitelisted manually.
 --
+
+== Master organization and business groups
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
== Master Organization and Business Groups
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 2]
```diff
@@ -47,3 +47,11 @@ Apps that remove scopes do not need to be whitelisted again.
...
+
+Connected Apps are at the Master Organization level, not at the Business Group(Sub Organization) levels.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
By default, connected apps are configured at the master organization level, not at the business group (sub-organization) levels.
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 3]
```diff
@@ -47,3 +47,11 @@ Apps that remove scopes do not need to be whitelisted again.
...
+
+If a connected app is created to act on its own behalf(client credentials), after the scopes scopes are chosen, you could choose which master ogranization/Business Groups to enable it.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
When you create a connected app to act on its own behalf using client credentials, you can choose which master organization or business groups to enable it in after you choose scopes.
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 4]
```diff
@@ -47,3 +47,11 @@ Apps that remove scopes do not need to be whitelisted again.
...
+
+If a connected app is created to act on behalf of a user, what the connected app can do is based on the scopes and what organization/business groups this user has access to.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
A connected app that acts on behalf of a user receives its own access and permissions based on the scopes, organizations, and business groups to which the user has access.
```

**Review by sarathecoubian — DISMISSED** [review]
Add suggestions; will merge and approve

**Review by dukesphere — APPROVED** [review]

**Pull request merged**